exporter/awsprometheusremotewriteexporter/auth_test.go

```diff
@@ -16,6 +16,7 @@ package awsprometheusremotewriteexporter
 
```

Comment:
If I'm not mistaken we don't have a test verifying the user agent (it would be some sort of fuzzy match I guess)

Reply:
I have modified the tests to add verification of the user-agent header